### ISSUE TITLE
Nodester applicaton not starting and no logs

### ISSUE THREAD

**Issue by jtwebman** [nodester/nodester#212]
I created a application this morning called googleplus2rss and I can push to the git and get info but it never starts and I get no log ether. See commands:

$ nodester app start googleplus2rss
nodester info Attemping to start app: googleplus2rss
nodester info app started.

Though I was getting earlier: nodester info app started.

$ nodester app info googleplus2rss
nodester info Gathering information about: googleplus2rss
nodester info googleplus2rss on port 11825 running: failed-to-start (pid: unknown)
nodester info gitrepo: git@nodester.com:/node/git/jtwebman/3672-ef78e464596020a6bce989932b9021dd.git
nodester info appfile: googleplusrss.js

$ nodester app logs googleplus2rss
nodester ERROR [HTTP 500] failure

My code for this is up on Github here as well and runs locally with no issues: https://github.com/jtwebman/GooglePlusToRSSFeed

Sorry about the repeat but Amurray closed my item though it still is not working.

**Comment by funkytek** [comment]
Can you try creating a new app for comparison?

**Comment by jtwebman** [comment]
Ok I deleted my starter application and re-created it and it works even though I did get a HTTP 408 when it tried to start the app on init after I just created it. See below:

$ nodester app init jtwebman
nodester info initializing git repo for jtwebman into folder jtwebman
nodester info cloning the repo git clone git@nodester.com:/node/git/jtwebman/3754-3ba96823a165be92a0504f0712efd1e9.git jtwebman
nodester info clone complete
nodester info Writing app data to config in jtwebman/.nodester.appconfig
nodester info writing app files
nodester info processing the initial commit
nodester info attemping to start the new app.
nodester ERROR [HTTP 408] failure - application already running.

I'll remove and try the other application now.
